processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorSubscription.java

```diff
@@ -229,7 +233,9 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
                 pollOnce(consumer);
                 if (System.currentTimeMillis() - lastCommittedMillis >= commitIntervalMillis.value()) {
                     try {
+                        Timer timer = Utils.timer();
                         commitCompletedOffsets(consumer);
+                        metrics.commitOffsetTime.record(timer.duration());
```

Comment:
- This should be recorded outside of `if` block I think, because `SubscriptionMetrics` sounds like "the breakdown of each consuming loop" (as description of the metrics)
- And if we count this inside `if` block, I guess 99%ile and 99.9%ile value cannot be measured correctly since expiry is set to 60sec but commit interval is 1000ms by default

Reply:
Ah that's very nice catch, yes indeed, fixing, thx.